src/components/App.js

```diff
@@ -53,9 +82,16 @@ class AppElement extends LitElement {
         <nav>
           <h1><a href="/">Swapi</a></h1>
           <ul>
-            <li>
-              <a href="/people">People</a>
-            </li>
+            ${
+              Object.entries(this._topics).map(([topic]) => 
```

Comment:
Creo que este queda mejor con lo que se está haciendo
```suggestion
              Object.keys(this._topics).map((topic) => 
```

Reply:
Chevere, gracias :)